features/step_definitions/policy_area_steps.rb

```diff
@@ -17,6 +17,14 @@
   create_policy_area(name: policy_area_name)
 end
 
+When(/^I associate the policy area with an organisation$/) do
+  visit policy_areas_path
+  click_on @policy_area.name
+
+  select 'Organisation 1', from: "Organisations"
+  click_on "Save"
+end
```

Comment:
We've been moving towards an abstraction here where steps don't directly prod the page like this.  See https://github.com/alphagov/policy-publisher/blob/master/features/step_definitions/policy_area_steps.rb#L8-L18